geopandas/tests/test_geom_methods.py

```diff
@@ -1494,6 +1477,7 @@ def test_explode_order(self):
         expected_index = MultiIndex.from_arrays(
             [[2, 2, 9, 9, 7, 7], [0, 1, 0, 1, 0, 1]],
         )
+        assert (expected_index.dtypes == "int64").all()
```

Comment:
If this works, I'll remove this and the change below, I'm just explicitly checking what I think happens on linux is actually true according to ci)